graph/vertex/impl/ThingVertexImpl.java

```diff
@@ -246,7 +241,7 @@ public boolean isWrite() {
 
         public static class Buffered extends ThingVertexImpl.Write {
 
-            protected boolean isInferred;
+            private final boolean isInferred;
```

Comment:
`protected` was a mistake before, should have been private!
Additionally, we can now use `final`, since the inferred status is not required to change at all